tests/test_label_categories.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import pytest
+
+from datumaro import LabelCategories
+
+from tests.requirements import Requirements, mark_requirement
+
+
+@mark_requirement(Requirements.DATUM_GENERAL_REQ)
+def test_add_category():
+    categories = LabelCategories()
+    index = categories.add("cat")
+    assert index == 0
+    assert len(categories) == 1
+    assert categories[0].name == "cat"
+
+
+@mark_requirement(Requirements.DATUM_GENERAL_REQ)
+def test_add_category_with_parent():
```

Comment:
Please rename tests to use naming convention: `test_can_...`

Reply:
Done!